src/python/sense/client/requestclient.py

```diff
@@ -0,0 +1,48 @@
+import json
+import requests
+from sense.common import evalInput
+from sense.client.mainclient import MainClient
+
+class RequestClient(MainClient):

```

Comment:
RequestClient is a very generic name. I would suggest renaming to: RequestWrapper.